common/hardcoded_params/include/hardcoded_params/control_limits/control_limits.h

```diff
@@ -21,9 +21,19 @@ namespace hardcoded_params
 namespace control_limits
 {
 /**
- * The maximum allowable longitudinal velocity of the vehicle in m/s 
+ * The maximum allowable longitudinal velocity of the vehicle in m/s
  */
 constexpr double MAX_LONGITUDINAL_VELOCITY_MPS = 35.7632;
 constexpr double MAX_LONGITUDINAL_ACCEL_MPS2 = 3.5;
+
+/**
+* The maximum allowable simulation parameters
+* At the time of writing this, platform uses CARLA 0.9.10 as the vehicle's physics simulator.
+* CARLA's this version uses below hardcoded value for its vehicles (so even if increasing this value,
```

Comment:
Minor typo: Maybe change `CARLA's this version` to `CARLA 0.9.10`?

Reply:
Edited